atom/browser/api/atom_api_session.cc

```diff
@@ -169,9 +171,9 @@ void OnGetBackend(disk_cache::Backend** backend_ptr,
   if (result != net::OK) {
     RunCallbackInUI(callback, result);
   } else if (backend_ptr && *backend_ptr) {
-    (*backend_ptr)->DoomAllEntries(base::Bind(&RunCallbackInUI, callback));
+    (*backend_ptr)->DoomAllEntries(base::Bind(&RunCallbackInUI<int>, callback));
```

Comment:
Hmm it might be needed when passing it to `base::Bind`, but when calling it directly the `<int>` should not be needed.